android-patches/patches-droid-office-grouped/OfficeRNHost/React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -1,10 +1,10 @@
---- "e:\\github\\fb-react-native-forpatch-base\\React\\CxxBridge\\RCTCxxBridge.mm"	2020-01-30 13:55:48.476581100 -0800

```

Comment:
I reverted this to head and am not seeing a diff when I view the before/after in a text editor but it's still appearing here. @tom-un any idea what's going on with the patches file?

Reply:
Its most likely just LF versus CRLF line ending differences.   I tweaked the gitattributes to be always LF for .mm files.   It should be harmless

Reply:
I was more curious why it would randomly show up but agree that it's totally harmless

Reply:
So it sounds like we should commit this file one-off (separate out from this PR) and it will be fixed for good?

Reply:
Sure. PR is here https://github.com/microsoft/react-native-macos/pull/364